### PR TITLE
modify_resolve_function

### PR DESCRIPTION
**Pull request by sxhthreo**
modified:   examples/scripts/possible_paths.py

## Summary by CodeRabbit

- **Refactor**
	- Modified the method of retrieving specific functions in scripts to enhance accuracy.

### CODE REVIEW

**Comment by 0xalpharush** [comment]
Would you sign the CLA and run `make reformat` please?

File: examples/scripts/possible_paths.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def resolve_function(contract_name, function_name):
     contract = contracts[0]
     # Obtain the target function
     target_function = next(
-        (function for function in contract.functions if function.name == function_name), None
+        (function for function in contract.functions_declared if function.name == function_name), None
```
[the remark this answers is not in the document]
Comment (sxhthreo) [reply examples/scripts/possible_paths.py:22]: yes

[the remark this answers is not in the document]
Comment (sxhthreo) [reply examples/scripts/possible_paths.py:22]: `make reformat`

**Comment by sxhthreo** [comment]
> Would you sign the CLA and run `make reformat` please?

Sorry, after I signed the CLA, I DON'T know what to do next.

**Comment by 0xalpharush** [comment]
Thanks for fixing! I ran the formatter and merged in https://github.com/crytic/slither/pull/2312

**Pull request closed without merging**